### PR TITLE
run spdx-correct in the summarizers

### PR DESCRIPTION
**Pull request by dabutvin**

### CODE REVIEW

File: providers/summary/clearlydefined.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ class ClearlyDescribedSummarizer {
         'licensed.declared',
         license
           .split('/')
-          .map(normalizeSpdx)
+          .map(correct)
```
Comment (jeffmcaffer) [inline providers/summary/clearlydefined.js]: are you going to pass in the option to turn off upgrades?

I noticed this being called in multiple places. If we care about the params being passed in, should we have a helper method that everyone uses to ensure consistency? Perhaps revive `normalizeSPDX` but have it just do `correct` with the designated args.

File: providers/summary/scancode.js [hunk 2]
```diff
@@ -82,7 +83,7 @@ class ScanCodeSummarizer {
 
   _toExpression(licenses) {
     if (!licenses) return null
-    const list = setToArray(licenses)
+    const list = setToArray(licenses).map(correct)
```
Comment (jeffmcaffer) [inline providers/summary/scancode.js]: Similarly here, turn off upgrades? or is that the default?

File: providers/summary/fossology.js [hunk 3]
```diff
@@ -28,7 +29,8 @@ class FOSSologySummarizer {
     return files
       .map(file => {
         const path = get(/^File (.*?) contains/.exec(file), '[1]')
-        const license = normalizeSpdx(get(/license\(s\) (.*?)$/.exec(file), '[1]'))
+        let license = get(/license\(s\) (.*?)$/.exec(file), '[1]')
+        if (license) license = correct(license)
```
Comment (jeffmcaffer) [inline providers/summary/fossology.js]: and here

File: providers/summary/scancode.js (hunk 2 above)
Comment (dabutvin) [reply providers/summary/scancode.js]: Ahh. Duh. Thanks for finding

**Pull request merged**